set_tags/utils.py

```diff
@@ -126,3 +129,51 @@ def get_synapse_tags(synapse_id):
   tags = list(synapse_user_tags + synapse_team_tags)
   log.debug(f'Synapse tags: {tags}')
   return tags
+
+def get_provisioned_product_name_tag(tags):
+  '''Get provisioned product name among the resource tags.
+  :param tags: the list of tags on the instance, assume to contain a provisioned product
+         ARN tag that is applied by AWS and it's value should be in the following format
```

Comment:
just being anal:  "it's" should be "its"

Reply:
fixed